rules/docker-telemetry.mk

```diff
@@ -29,6 +29,9 @@ endif
 
 $(DOCKER_TELEMETRY)_CONTAINER_NAME = telemetry
 $(DOCKER_TELEMETRY)_RUN_OPT += --privileged -t
+$(DOCKER_TELEMETRY)_RUN_OPT += --memory 450m
```

Comment:
Please make sure this scenario does not happen:
1. the memory_checker seems running inside telemetry docker, and the threshold is 400m: https://github.com/sonic-net/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/base_image_files/monit_telemetry

2. if memory_checker run first time check and current memory is 390m, memory_checker passed. memory_check will check 1 minute later.

3.  then before memory next time run, telemetry use more than 450M memory.

4. because can't allocate memory, memory_checker failed. telemetry container can't start again, telemetry container will hang there and memory will never release.

https://docs.docker.com/engine/reference/run/#user-memory-constraints

Reply:
With https://github.com/sonic-net/sonic-buildimage/pull/19179, we're using monit to monitor runtime memory usage of docker, @qiluo-msft do we still need this docker hard limit as double insurance?